------------------------------- INFO --------------------------------
|
|  Online:
|     online model path:
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/model/online/yolov3.pth,
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/model/online/yolov3_int8.pth
|     accuracy test code position: 
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/online/yolov3/test.py.
|
|  Offline:
|     offline model path: 
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/model/offline/yolov3.cambricon
|     accuracy test code position: 
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/offline/yolov3/run_offline_accuracy.sh 
|         /workspace/volume/private/CK_TEST/00_Yolov3_example/result/offline/scripts/meanAP_COCO.py.
|
------------------------------- RUN ---------------------------------
Running online.
/workspace/volume/private/CK_TEST/00_Yolov3_example/online/yolov3 /workspace/volume/private/CK_TEST/00_Yolov3_example/result
CNML: 7.10.2 0a592c0
/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torch/nn/_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='mean' instead.
  warnings.warn(warning.format(ret))
CNRT: 4.10.1 a884a9a
Traceback (most recent call last):
  File "test.py", line 185, in <module>
    nms_thres=0.5)
  File "/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torchvision/models/quantization/object_detection/yolov3/models.py", line 41, in yolov3
    state_dict = torch.load(quantized_path)
  File "/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torch/serialization.py", line 500, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '../../model/online/yolov3_int8.pth'
/workspace/volume/private/CK_TEST/00_Yolov3_example/result
Running offline.
/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3 /workspace/volume/private/CK_TEST/00_Yolov3_example/result
CNML: 7.10.2 0a592c0
/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torch/nn/_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='mean' instead.
  warnings.warn(warning.format(ret))
CNRT: 4.10.1 a884a9a
Traceback (most recent call last):
  File "test.py", line 185, in <module>
    nms_thres=0.5)
  File "/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torchvision/models/quantization/object_detection/yolov3/models.py", line 41, in yolov3
    state_dict = torch.load(quantized_path)
  File "/workspace/volume/private/sdk/venv/pytorch/lib/python3.7/site-packages/torch/serialization.py", line 500, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '../../model/online/yolov3_int8.pth'
/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3

====================================================
running yolov3 offline multiple core ...
----------------------
running offline test...
run_cmd: /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../build/yolov3/src/yolov3_offline_multicore -offlinemodel /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../../model/offline/yolov3.cambricon -images /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/data/file_list_for_release -labels /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/label_map_coco.txt -outputdir /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/output -dump 1 -simple_compile 1 -dataset_path /workspace/dataset/public/zhumeng-dataset/coco_2014/  -input_format 0 &>> /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/yolov3_multicore_thread.log
check_cmd: python /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../scripts/meanAP_COCO.py --file_list /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/data/file_list_for_release --result_dir /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/output --ann_dir /workspace/dataset/public/zhumeng-dataset/coco_2014/ --data_type val2014 &>> /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/yolov3_multicore_thread.log
run_offline_accuracy.sh: line 43: /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../build/yolov3/src/yolov3_offline_multicore: No such file or directory

run_cmd: /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../build/yolov3/src/yolov3_offline_multicore -offlinemodel /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../../model/offline/yolov3.cambricon -images /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/data/file_list_for_release -labels /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/label_map_coco.txt -outputdir /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/output -dump 1 -simple_compile 1 -dataset_path /workspace/dataset/public/zhumeng-dataset/coco_2014/  -input_format 0 &>> /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/yolov3_multicore_thread.log
check_cmd: python /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../scripts/meanAP_COCO.py --file_list /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/data/file_list_for_release --result_dir /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/output --ann_dir /workspace/dataset/public/zhumeng-dataset/coco_2014/ --data_type val2014 &>> /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/yolov3_multicore_thread.log
run_offline_accuracy.sh: line 43: /workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../build/yolov3/src/yolov3_offline_multicore: No such file or directory
Traceback (most recent call last):
  File "/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../scripts/meanAP_COCO.py", line 172, in <module>
    img_ids, res_file = generate_json_file(img_list, result_dir, class_dict, json_file)
  File "/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../scripts/meanAP_COCO.py", line 124, in generate_json_file
    result_objs = parse_output(result_dir + '/' + img_name + '.txt')
  File "/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/../scripts/meanAP_COCO.py", line 90, in parse_output
    f = open(input)
FileNotFoundError: [Errno 2] No such file or directory: '/workspace/volume/private/CK_TEST/00_Yolov3_example/offline/yolov3/output/COCO_val2014_000000000164.txt'
/workspace/volume/private/CK_TEST/00_Yolov3_example/result
